### ISSUE TITLE
Cant type db url in console

### ISSUE THREAD

**Issue by Elufimov** [poxet/Influx-Capacitor#23]
Hi! I try setup server from console ` .\Tharga.Influx-Capacitor.Console.exe server config` but i simply cant type `:` . When paste url from buffer it appears trimmed. `http://localhost` become `http//localhost` 

Microsoft Windows Server 2012 R2 Standard

P.S. Can you provide example of server config? I will create it manually ;)

**Comment by poxet** [comment]
I will have a look at the issue with typing :

The file should typically be located here:
C:\ProgramData\Thargelion\Influx-Capacitor\database.xml

And look like this...
<?xml version="1.0" encoding="utf-16"?>
<Influx-Capacitor>
  <Database>
    <Url>http://domain:8086</Url>
    <Username>x</Username>
    <Password>x</Password>
    <Name>x</Name>
  </Database>
</Influx-Capacitor>

(Name is the name of the Influx database and it must exist for this to work.)

When creating this from the console the password will be encrypted, but the service can read unencrypted passwords as well.

You should be able to test the configuration from the console by typing "config show". Then you can see the configuration and see if the database can be accessed.

**Comment by poxet** [comment]
What language are you running windows on? (Helps when trying to find the ':' issue...)

**Comment by Elufimov** [comment]
@poxet 
```
PS C:\> Get-Culture

LCID             Name             DisplayName
----             ----             -----------
1049             ru-RU            Russian (Russia)
```

**Comment by Elufimov** [comment]
@poxet I can run any command that can help whit debug this issue.

**Comment by nathanwebb** [comment]
Same issue here - brand new install on a windows 7 client.

running 1.0.14. Language is en-AU.  Colon works perfectly fine in the terminal or powershell, just not in the influx-capacitor console.

**Comment by poxet** [comment]
I think the issue is with a nuget package that I am using. Will have a look at that as soon as possible.

**Comment by davideuler** [comment]
I've met exactly the same problem. Help wanted.

**Comment by davideuler** [comment]
Solved the problem by the workaround:
create a database.xml file here: C:\ProgramData\Thargelion\Influx-Capacitor\database.xml, and drop the content poxet mentioned.
Note, C:\ProgramData is a hidden directory.

Then it works.

**Comment by poxet** [comment]
As of version 1.0.16 this should work. Please get back to me if it does not.

**Comment by switzer60** [comment]
Installed on Windows 2012 R2 Standard Edition today.  The problem persists.  Culture and Locale are 1033

**Comment by poxet** [comment]
Okey, I am reopening the issue...

What version of Influx-Capacitor are you running?

**Comment by switzer60** [comment]
1.0.16

http://influx-capacitor.com/Resources/Production/Influx-Capacitor.1.0.16.msi

**Comment by poxet** [comment]
I have created a debug-log for this case. When a button is pressed in the console the keycode recieved is written to the file C:\Temp\Tharga.Influx-Capacitor.Console.txt

Please install version 1.0.17.99 and try it out.
http://influx-capacitor.com/Home/Download

Try typing ':' and check if there is an entry in the logfile. Post the result here and I will try to fix it once and for all.

**Comment by switzer60** [comment]
Logged in as Local Administrator

Fails to install:
Application PopUp: Influx Capacitor Setup
Service 'Influx-Capacitor' (Influx-Capacitor) failed to start. Verify that you have sufficient privileges to start system services.

Windows System Event Log Reports:
Event ID 26
Application popup: Tharga.Influx-Capacitor.Services.exe - Application Error : The exception unknown software exception (0xe0434352) occurred in the application at location (0x76e35b68).

If I Ignore the error (Cancel, Retry, Ignore) (and complete the install) and launch the Influx-Console, I get the following error: 
Unhandled Exception: System.IO.FileNotFoundException: Could not load file or assembly 'log4net, Version 1.2.15.0, Culture=neutral, PublicKeyToken=669xxxxxxxxxxa2a' or one of its dependencies. The system cannot find the file specified.  
   at Tharga.Influx_Capacitor.MyLogger..ctor()
   at Tharga.InfluxCapacititor.Console.CompositeRoot..ctor(
   at Tharga.InfluxCapacitor.Console.Program.Main(String[] args)

I hand transposed these error messages, typos possible.

**Comment by poxet** [comment]
Ah, sorry. I forgot to update the msi to include the log4net component. Ill fix as soon as possible and test the installation too this time.

**Comment by poxet** [comment]
Yet another try, version 1.0.17.103 and newer should work to install with debugging. Please get back to me with the input of file C:\Temp\Tharga.Influx-Capacitor.Console.txt after typing :

**Comment by poxet** [comment]
Now version 1.0.17 with debugging disabled is put in production. To get the correct logging, enable the content of the log4net section in the console config file, make sure that you have write permissions to the path and change the <level value="WARN" /> to <level value="ALL" />

Then the keys pressed will be logged. If you still have issues with the : sign, please send the log and I will resume investigation.

**Comment by switzer60** [comment]
Issue appears to be resolved for me in version 1.0.17 (stable)
